requirements.txt

```diff
@@ -4,6 +4,7 @@ pip>=22.2.0
 bidict~=0.22.1
 blinker~=1.5
 extendparser~=0.3.1
+importlib-resources
```

Comment:
are you sure? `importlib.resources` is in STL now, and we PrusaLink use native STL package.